README.md

```diff
@@ -67,25 +67,60 @@ For a full list of available tools please see the [tools section](http://fulcrum
 
 Below we highlight a few tools that you may find useful.
 
-* Tools for working with Unique Molecular Indexes (UMIs, aka Molecular IDs or MIDs). 
-  * Annotating/Extract Umis from read-level data: `AnnotateBamWithUmis` and `ExtractUmisFromBam`.
-  * Tools to manipulate read-level data containing Umis: `CorrectUmis`, `GroupReadsByUmi`, `CallMolecularConsensusReads` and `CallDuplexConsensusReads`
-* Tools to manipulate read-level data:
-  * FastqManipulation: `DemuxFastqs` and `FastqToBam`
-  * Filter read-level data: `FilterBam`.
-  * Clipping of reads: `ClipBam`.
-  * Randomize the order of read-level data: `RandomizeBam`.
-  * Update read-level metadata: `SetMateInformation` and `UpdateReadGroups`.
-* Quality assessment tools:
-  * Detailed substitution error rate evaluation: `ErrorRateByReadPosition`
-  * Sample pooling QC: `EstimatePoolingFractions`
-  * Splice-aware insert size QC for RNA-seq libraries: `EstimateRnaSeqInsertSize`
-  * Assessment of duplex sequencing experiments: `CollectDuplexSeqMetrics`
-* Miscellaneous tools:
-  * Pick molecular indices (ex. sample barcodes, or molecular indexes): `PickIlluminaIndices` and `PickLongIndices`.
-  * Convert the output of HAPCUT (a tool for phasing variants): `HapCutToVcf`.
-  * Find technical or synthetic sequences in read-level data: `FindTechnicalReads`.
-  * Assess phased variant calls: `AssessPhasing`.
+-   Tools for working with Unique Molecular Indexes (UMIs, aka Molecular IDs or Molecular Barcodes):
+    -   Annotating/Extract Umis from read-level data: [`FastqToBam`][fgbio-fastqtobam-link], [`AnnotateBamWithUmis`][fgbio-annotatebamwithumis-link], [`ExtractUmisFromBam`][fgbio-extractumisfrombam-link], and [`CopyUmiFromReadName`][fgbio-copyumifromreadname-link].
+    -   Manipulate read-level data containing Umis: [`CorrectUmis`][fgbio-correctumis-link], [`GroupReadsByUmi`][fgbio-groupreadsbyumi-link], [`CallMolecularConsensusReads`][fgbio-callmolecularconsensusreads-link], [`CallDuplexConsensusReads`][fgbio-callduplexconsensusreads-link], and [`FilterConsensusReads`][fgbio-filterconsensusreads-link].
```

Comment:
```suggestion
-   Tools for working with Unique Molecular Indexes (UMIs, aka Molecular IDs or Molecular Barcodes):
    -   Annotating/Extract UMIs from read-level data: [`FastqToBam`][fgbio-fastqtobam-link], [`AnnotateBamWithUmis`][fgbio-annotatebamwithumis-link], [`ExtractUmisFromBam`][fgbio-extractumisfrombam-link], and [`CopyUmiFromReadName`][fgbio-copyumifromreadname-link].
    -   Manipulate read-level data containing UMIs: [`CorrectUmis`][fgbio-correctumis-link], [`GroupReadsByUmi`][fgbio-groupreadsbyumi-link], [`CallMolecularConsensusReads`][fgbio-callmolecularconsensusreads-link], [`CallDuplexConsensusReads`][fgbio-callduplexconsensusreads-link], and [`FilterConsensusReads`][fgbio-filterconsensusreads-link].
```

Although UMI gets title cased in the tool names, I think it should be all caps in the docs (since it's an acronym)

Reply:
https://www.youtube.com/watch?v=NPiAK33CfxQ